hv.c

```diff
@@ -4120,6 +4125,21 @@ Perl_cop_store_label(pTHX_ COP *const cop, const char *label, STRLEN len,
         = refcounted_he_new_pvs(cop->cop_hints_hash, ":", labelsv, 0);
 }
 
+/* share doesn't mean "shared" but "shared string table" */
+/* TODO: should this be atomic increment ??? */
+const struct hek *
+share_hek_hek (const struct hek *hek)
+{
+    ++(
```

Comment:
I'd get rid of these parentheses and use postfix increment or even `+= 1`.

Reply:
parentheses are here only because they were there in original macro.
in long term I have a gut feeling that something like `ATOMIC_INCREMENT` macro will be more suitable here (but I'm not much familiar with its usage yet so letting it as it is)

Reply:
899a2b3